### DISCUSSION TITLE
Generally prefer source packages

### DISCUSSION THREAD

**Discussion by staalmannen** [BioArchLinux/Packages#7] [Q&A]
An example:

I noticed that the ugene package is the ugene-bin package. While convenient, it comes with some drawbacks. It comes bundled with a lot of libraries that otherwise could be provided by the system (as is the case when compiled from source). The package is 1GiB when installed (!). In comparison, the from source compiled version is much smaller. The from source version is also following the system theme.

**Comment by michaelgelliot** [comment]
I agree with this preference!
